docs/scos/dev/the-docker-sdk/202212.0/configuring-debugging-in-docker.md

```diff
@@ -210,3 +210,9 @@ Find several examples below:
 * `docker/sdk cli -x console queue:worker:start`
 * `docker/sdk console -x queue:worker:start`
 * `docker/sdk testing -x codecept run -codeception.yml`
+
+The [PHPMD](https://github.com/phpmd/phpmd/blob/master/src/bin/phpmd#L29) command requires the `PHPMD_ALLOW_XDEBUG` env variable for debug mode:
+```
```

Comment:
define the code block type—for example:
```suggestion
```php
```

Reply:
this is not php code. this is bash. what "code" type should I use?